tests/Tests/Cluster/RootNodeInfo/RootNodeInfoApiTests.cs

```diff
@@ -62,22 +62,13 @@ protected override void ExpectResponse(RootNodeInfoResponse response)
 			response.Version.Should().NotBeNull();
 			response.Version.Number.Should().NotBeNullOrWhiteSpace();
 			response.Version.BuildDate.Should().BeAfter(default);
-			if (TestConfiguration.Instance.ServerType == OpenSearch.Stack.ArtifactsApi.ServerType.OpenSearch)
-				response.Version.Distribution.Should().NotBeNullOrWhiteSpace();
-			else
-				response.Version.BuildFlavor.Should().NotBeNullOrWhiteSpace();
```

Comment:
I think you can remove `BuildFlavor` from `NodeInfo` and `OpenSearchVersionInfo` too.